LICENSE.txt

```diff
@@ -2,6 +2,9 @@ The MIT License (MIT)
 
 Copyright (c) 2015 Formidable Labs
 
+Copyrights are respective of each contributor listed at the beginning of each
+typescript definition file.
+
```

Comment:
I took this language from the DefintelyTyped license (we did a similar addition to our license for `nuka` when taking the definitely typed definition files).

Reply:
Perfect, thank you!